pom.xml

```diff
@@ -11,7 +11,7 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.1.7.RELEASE</version>
+		<version>2.7.5</version>
```

Comment:
So just an FYI, the catalog will be bumping to 3.0.6 for sprint boot with new security config stuff.

Thoughts on going ahead and bumping this as well?

Completely understand if you'd rather we get these changes in and then save that refactor for another issue.
I haven't got the catalog updates in yet so I'm perfectly fine with waiting till those are in to come back and refactor the ML service.

Reply:
Just tried it and there was a problem with the Java version on the docker image and 3.0.6, so let's make it a separate issue.